packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/DocumentController.java

```diff
@@ -26,19 +29,27 @@
 import java.nio.charset.StandardCharsets;
 
 import java.io.IOException;
+import java.io.InputStream;
 import java.util.List;
 
 @RequestMapping("/document-asset")
 @RestController
 @Slf4j
 public class DocumentController implements SnakeCaseController {
 
+	private final DownloadService downloadService;
 	@Autowired
 	DocumentProxy proxy;
 
 	@Autowired
 	JsDelivrProxy gitHubProxy;
 
+
```

Comment:
Similar here, this could be just autowired directly like the JsDelivrProxy, etc, is